platforms/documentation/docs/src/docs/userguide/jvm/groovy_plugin.adoc

```diff
@@ -177,7 +177,7 @@ _Default value_: Not null
 +
 All Groovy source files of this source set. Contains only the `.groovy` files found in the Groovy source directories.
 
-These properties are provided by a convention object of type link:{groovyDslPath}/org.gradle.api.tasks.GroovySourceSet.html[GroovySourceSet].
+These properties are provided by a convention object of type link:{groovyDslPath}/org.gradle.api.tasks.SourceSet.html[GroovySourceSet].
```

Comment:
❌ The link text no longer matches the target `GroovySourceSet` => `SourceSet`.

Also the deprecation of `GroovySourceSet` seems complicated, maybe the surrounding documentation needs updates as well?

> Using convention to contribute to source sets is deprecated. You can configure the groovy sources via the `GroovySourceDirectorySet` extension (e.g.
> `sourceSet.getExtensions().getByType(GroovySourceDirectorySet.class).setSrcDirs(...)`). This interface is scheduled for removal in Gradle 9.0.

Reply:
These pages are due for an update in a future PR.